ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -30,7 +48,7 @@ prefix_mask_packages: |
   >=sys-libs/glibc-2.34
   # stick to GCC 9.x; using a too recent compiler in the compat layer complicates stuff in the software layer,
   # see for example https://github.com/EESSI/software-layer/issues/151
-  >=sys-devel/gcc-10
+  >=sys-devel/gcc-10.4.1
```

Comment:
Comment above should be updated accordingly?
But maybe keep the pointer to https://github.com/EESSI/software-layer/issues/151 since that can help explain why we prefer sticking to an older GCC version.

@trz42 Did you try to build `GCC/9.3.0` on top of this compat layer with GCC 10.x as system compiler?

Reply:
Good catch about the comment. We should keep the pointer yes.

Yes, I've built GCC/9.3.0 and also then software. See https://github.com/trz42/software-layer/pull/42

Reply:
Updated comment.